src/app/centers/centers.component.html

```diff
@@ -29,9 +29,11 @@
       <ng-container matColumnDef="status">
         <th mat-header-cell *matHeaderCellDef mat-sort-header='status'> Status </th>
         <td mat-cell *matCellDef="let row"> <div [className]="row.active === true ? 'active' : 'inactive'">
-            <i class="fa fa-circle fa-2x"> </i>
+            <mat-icon matListIcon>
```

Comment:
Please update all icons in table in a similar manner to how its being used in `chart-of-accounts.component.ts` of the accounting section. Here is an example for your reference:
```html
<div [className]="row.active === true ? 'active' : 'inactive'">
    <fa-icon matTooltip="{{ row.active === true ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="circle" size="lg"></fa-icon>
</div>
```